animations-tutorial.md

```diff
@@ -0,0 +1,589 @@
+                        duration: const Duration(milliseconds: 500));
+final Animation curve =
+    new CurvedAnimation(parent: animation, curve: Curves.easeOut);
+Animation<int> alpha = new IntTween(
+  begin: 0,
+  end: 255
+).animate(curve);
+```
+
+You can also chain Tweens together using the
+[`chain`](http://docs.flutter.io/flutter/animation/Animatable/chain.html)
+method, in which case a single `Animation` object will be configured by multiple
+`Tween` objects called in succession. This is different than calling `.animate`
+twice, in which case you have two Animate objects, each configured with a
+single Tween.
+
+## Animation notifications
+
+An Animation object can have Listeners and StatusListeners, defined with
+[`addListener`](http://docs.flutter.io/flutter/animation/Animation/addListener.html) and
+[`addStatusListener`](http://docs.flutter.io/flutter/animation/Animation/addStatusListener.html).
+A Listener is called whenever the value of the animation changes. The most common behavior of a Listener is to call setState() to cause a rebuild. A StatusListener is called when an animation begins, ends, moves forward, or moves reverse, as defined by [`AnimationStatus`](http://docs.flutter.io/flutter/animation/AnimationStatus-class.html).
+
+We’ll use `addListener` in the next section and we’ll use `addStatusListener` in Monitoring the progress of the animation.
+
+## Rendering animations
+
+So far all we’ve done is to generate a sequence of numbers over time. You’ll notice we haven’t actually rendered anything.
+
+To render with an `Animation<>` object, store the `Animation<>` object as a member of your `Widget`, then use its value to decide how to draw.
+
+Consider this application that draws the Flutter logo without animation:
+
+```dart
+import 'package:flutter/material.dart';
+
+const kLogoUrl = 'https://raw.githubusercontent.com/dart-lang/logos/master/logos_and_wordmarks/dart-logo.png';
+
+class LogoApp extends StatefulComponent {
+  LogoAppState createState() => new LogoAppState();
+}
+
+class LogoAppState extends State<LogoApp> {
+
+  Widget build(BuildContext context) {
+    return new Center(
+        child: new Container(
+            margin: new EdgeDims.symmetric(vertical: 10.0),
+            height: 300.0,
+            width: 300.0,
+            child: new NetworkImage(src: kLogoUrl)));
+  }
+}
+
+void main() {
+  runApp(new LogoApp());
+}
+```
+
+This listing shows the changes necessary to animate the logo to grow outward: (Changes noted in bold)
+
+<pre><code class="language-dart">
+<b>import 'package:flutter/animation.dart';</b>
+import 'package:flutter/material.dart';
+
+const kLogoUrl = "https://raw.githubusercontent.com/dart-lang/logos/master/logos_and_wordmarks/dart-logo.png";
+
+class LogoApp extends StatefulComponent {
+  LogoAppState createState() => new LogoAppState();
+}
+
+class LogoAppState extends State&lt;LogoApp&gt; {
+  Animation&lt;double&gt; animation;
+<b>
```

Comment:
Just a comment - I spent quite a while looking at the problem that we can have syntax highlighting or HTML, but not both. My conclusion after visually considering what I was asking of the reader was that the HTML boldface was more important than the code markup. It's inconsistent, but it's a lot easier to read, particularly in code examples where there are two or three new sections added. I don't think the decision should be made by looking at this markdown, because the intended result is purely visual.